Makefile

```diff
@@ -153,23 +153,41 @@ ifdef SKIP_DOCKER_BUILD
 ENVOY_INTEG_DEPS=noop
 endif
 
-all: dev-build
```

Comment:
It might be surprising for existing folks for the default make target (e.g. `make + [enter]`) do print out the help instead of doing what it currently does. We probably want to retain that behavior for now. Also, there might be other scripts/workflows that just call `make` which might start failing (maybe search and see if there are any?).

Reply:
sorry, that's my mistake, I'll move it to the bottom